scripts/build/termux_step_start_build.sh

```diff
@@ -25,6 +25,18 @@ termux_step_start_build() {
 		# "0" is the default revision, so only include it if the upstream versions contains "-" itself
 		TERMUX_PKG_FULLVERSION+="-$TERMUX_PKG_REVISION"
 	fi
+	# full format version for pacman
+	local TERMUX_PKG_VERSION_EDITED=${TERMUX_PKG_VERSION//-/.}
+	local INCORRECT_SYMBOLS=$(echo $TERMUX_PKG_VERSION_EDITED | grep -o '[0-9][a-z]')
+	if [ -n "$INCORRECT_SYMBOLS" ]; then
+		local TERMUX_PKG_VERSION_EDITED=${TERMUX_PKG_VERSION_EDITED//${INCORRECT_SYMBOLS:0:1}${INCORRECT_SYMBOLS:1:1}/${INCORRECT_SYMBOLS:0:1}.${INCORRECT_SYMBOLS:1:1}}
+	fi
+	local TERMUX_PKG_FULLVERSION_FOR_PACMAN="${TERMUX_PKG_VERSION_EDITED}"
+	if [ -n "$TERMUX_PKG_REVISION" ]; then
+		TERMUX_PKG_FULLVERSION_FOR_PACMAN+="-${TERMUX_PKG_REVISION}"
+	else
+		TERMUX_PKG_FULLVERSION_FOR_PACMAN+="-0"
+	fi
```

Comment:
I see here you doing same thing as in these places:
* https://github.com/termux/termux-packages/blob/2812170b4a5a9674b931b0dc3213064eac3cac4f/scripts/build/termux_step_create_pacman_package.sh#L39-L49
* https://github.com/termux/termux-packages/blob/2812170b4a5a9674b931b0dc3213064eac3cac4f/scripts/build/termux_create_pacman_subpackages.sh#L70-L81